docs/user-guide/slicing.ipynb

```diff
@@ -198,6 +201,76 @@
     "d['x', 1:2]['a'] == d['a']['x', 1:2]\n",
     "d['x', 1:2]['a'].coords['x'] == d.coords['x']['x', 1:2]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Slicing tools\n",
```

Comment:
I am not sure this is the best place for this documentation. As it was, the slicing page introduced the fundamental concepts essential for the understanding of how to use scipp, whereas this section is about specialized routines that are non-essential. Should this rather go into the documentation of these methods?

Reply:
Fine by me. So we leave the example in the 1D plotting, and then just rely on the sphinx documentation of the API?

Reply:
I suppose you can move the code examples to the sphinx docstrings?